index.js

```diff
@@ -27,6 +27,17 @@ var Schema = function(schemaDoc, schemaObj){
 	this.schemaObj = schemaObj;
 };
 
+/**
+ * Parse XML
+ *
+ * @param {string} sourcePath - path to xml document
+ * @returns The parsed Schema
+ */
+exports.parseXml = function(sourcePath, options) {
```

Comment:
This is not necessary, since you can use the underlying libxmljs, which is exported from this library.
See: line 13 of the very same index.js.